x-pack/plugin/core/src/main/resources/logs-mappings.json

```diff
@@ -9,6 +9,20 @@
               "type": "keyword"
             },
             "match_mapping_type": "string"
+          },
+          "match_ip": {
+            "match_mapping_type": "string",
+            "path_match":   "*.ip",
+            "mapping": {
+              "type": "ip"
+            }
+          },
+          "match_message": {
+            "match_mapping_type": "string",
+            "path_match":   "*.message",
+            "mapping": {
+              "type": "text"
```

Comment:
Not that it would save lots of space, but given that scoring doesn't help, we could disable norms.

```suggestion
              "type": "text",
              "norms": false
```

Reply:
Should we do the same for the top level `message` field?